### DISCUSSION TITLE
How do I find a log for the recognized application?

### DISCUSSION THREAD

**Discussion by Hainer1** [nextcloud/recognize#879] [Q&A] [answered]
I do not get a log entry from the application until the 15 minutes are up, how can I see if the application is running or paused for some reason? The application says that there should be a log entry after 15 minutes. How can I check the functionality? 

Objects are recognised correctly, but faces are not recognized after hours. 

I used NC26.0.2 and 4.1.1 recognized. I waiting on: .... 24 picures on queue...

**Comment by marcelklehr** [comment, marked as answer]
Make sure background jobs are enabled via cron.
